packages/oruga/src/components/autocomplete/Autocomplete.vue

```diff
@@ -56,7 +56,7 @@ const props = defineProps({
     /** Override existing theme classes completely */
     override: { type: Boolean, default: undefined },
     /** @model */
-    modelValue: { type: [String, Number], default: "" },
+    modelValue: { type: [String, Number], default: undefined },
```

Comment:
In TypeScript, it is generally not recommended to declare an object type using `any` for most situations. However, I have noticed this practice in Oruga's codebase multiple times, so it seems acceptable in this case 👍🏻 

```suggestion
    modelValue: { type: [String, Number, Object], default: undefined },
```